### PR TITLE
Update kivy to v1.9.1

### PR DESCRIPTION
**Pull request by crishoj**
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

**Comment by adidalal** [comment]
Why not Kivy-1.9.1-osx-python3.7z?

File: Casks/kivy.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-  url "https://kivy.org/downloads/#{version.split('-').first}/Kivy-#{version}-osx.dmg"
+  url "https://kivy.org/downloads/#{version.split('-').first}/Kivy-#{version}-osx-python2.7z"
   name 'Kivy'
   homepage 'http://kivy.org'
   license :mit
 
-  app 'Kivy.app'
-  binary 'Kivy.app/Contents/Resources/script', target: 'kivy'
+  app 'Kivy2.app', target: 'Kivy.app'
```
Comment (vitorgalvao) [inline Casks/kivy.rb]: [Not a good candidate for `:target`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

Comment (vitorgalvao) [inline Casks/kivy.rb]: Actually, it seems like [the developer recommends this renaming](https://kivy.org/docs/installation/installation-osx.html#installation-on-os-x).

**Comment by vitorgalvao** [comment]
@adidalal is correct. `3.7` should be here. If you want it for `2.7`, please submit it to [caskroom/versions](https://github.com/caskroom/homebrew-versions).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ff3e9c4a6623af44b8a071027e8dcf3f4edfc6d9. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**